optaplanner-website-docs/package-lock.json

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
Should this file package-lock.json be in git?
Check with Jiri - I believe this file should be gitignored.

Reply:
My `package-lock.json` gets modified when I run the build. I suspect it's because we don't have the `package.json`. When I created it, the build finished without modifying the lock file.

It seems as if the file doesn't work when there is no `package.json`. Should we add it?